db.js

```diff
@@ -1,20 +1,58 @@
-import {CosmosClient} from "@azure/cosmos";
+import { CosmosClient } from "@azure/cosmos";
```

Comment:
Do you wan't to include `@ts-check` here too? Maybe include a tsconfig file would help?